src/lib/db.ts

```diff
@@ -60,6 +63,7 @@ export interface OpenDocumentOptions {
   visibility?: "public" | "private";
   title?: string;
   groupUserConnections?: {};
+  originDoc?: string;
```

Comment:
I hadn't realized before we just mush every document field into the model here...seems messy